fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -2954,4 +2952,54 @@ public long loadDb(DataInputStream dis, long checksum) throws IOException, DdlEx
     public ConcurrentHashMap<Long, Database> getIdToDb() {
         return new ConcurrentHashMap<>(idToDb);
     }
+
+    private void createPartitionWithIndicesInternal(String clusterName, long dbId, long tableId, long baseIndexId,
```

Comment:
createPartitionWithIndicesElasticOnStorageMedium is a better name.

Reply:
Done